src/test/warning/warning-printer_test.ts

```diff
@@ -49,7 +60,7 @@ suite('WarningPrinter', () => {
 
   test('can handle printing no warnings', async() => {
     await printer.printWarnings([]);
-    assert.equal(output.toString(), '');
+    assert.deepEqual(output.toString(), '');
```

Comment:
for here and below: is this really necessary? Similar strings should have equality.